### PR TITLE
Prepare release 23.5

### PR DESCRIPTION
**Pull request by jaimergp**
## Description

✂️ snip snip ✂️ the making of a new release.

Xref https://github.com/conda/conda-libmamba-solver/issues/203

### CODE REVIEW

**Review by costrouc — APPROVED** [review]

File: .authors.yml [hunk 1]
```diff
@@ -34,16 +34,21 @@
...
-  num_commits: 1
+  num_commits: 3
   first_commit: 2022-03-04 23:25:33
   github: kenodegard
 - name: Albert DeFusco
   email: albert.defusco+gh@me.com
   num_commits: 1
   first_commit: 2023-03-01 13:29:51
   github: AlbertDeFusco
+- name: jakirkham
```
[the remark this answers is not in the document]
Comment (jezdez) [reply .authors.yml:50]:
```suggestion
- name: John Kirkham
```

File: .mailmap [hunk 2]
```diff
@@ -17,4 +17,5 @@ Jannis Leidel <jannis@leidel.info>
 Ken Odegard <kodegard@anaconda.com>
 Matthew R. Becker <beckermr@users.noreply.github.com>
 conda-bot <18747875+conda-bot@users.noreply.github.com> Conda Bot <18747875+conda-bot@users.noreply.github.com>
+jakirkham <jakirkham@gmail.com>
```
Comment (jezdez) [inline .mailmap:20]:
```suggestion
John Kirkham <jakirkham@gmail.com> jakirkham <jakirkham@gmail.com>
```

File: AUTHORS.md [hunk 3]
```diff
@@ -9,4 +9,5 @@ Authors are sorted alphabetically.
 * Ken Odegard
```
Comment (jezdez) [inline AUTHORS.md:9]:
```suggestion
* John Kirkham
* Ken Odegard
```

File: AUTHORS.md [hunk 4]
```diff
@@ -9,4 +9,5 @@ Authors are sorted alphabetically.
...
 * Matthew R. Becker
 * conda-bot
+* jakirkham
```
Comment (jezdez) [inline AUTHORS.md:12]:
```suggestion
```

**Review by jezdez — APPROVED** [review]
LGTM with comments

File: AUTHORS.md (hunk 3 above)
Comment (jaimergp) [reply AUTHORS.md:9]: John has chosen not to use his full name in his github profile so that's why we didn't amend this. Is that a problem copyright-wise?

Comment (kenodegard) [reply AUTHORS.md:10]: I don't have a legal answer, I just know that we've sought to use the contributor's full name whenever possible, it helps when trying to deduplicate.

**Pull request merged**